### ISSUE TITLE
Streaming to broad audience

### ISSUE THREAD

**Issue by QVDev** [meething/meething#106]
So for instances online classes or podcast recording can benefit from a couple of speakers and a bunch of watchers but dont want 30 people in the room.

So we should allow broadcasting the current meeting to a big audience. One thing is to have export options such as youtube live or twitch. But of course we have live streaming also working via gun network and can be improved in we take it into DAM communication 

This way you can have a panel of 5 people and than an audience of a billion people :) without sacrificing the quality and disturb the meeting.

**Comment by Dletta** [comment]
Yes and that was the Fitnessstudios or Cooking Lessons or Education use
case right there!

**Comment by QVDev** [comment]
And @lmangani had the great idea to also add audio message when questions are allowed in viewer mode

If your in a class broadcast you can directly record the question. Teacher wont be disrupted and end of class the teacher has all questions queued to answer them one by one or, can be text for that matter. Doesnt have to wait till end whenever teacher want it can be played.

**Comment by Dletta** [comment]
Very great idea and of course very marketable
